src/io/flutter/run/daemon/FlutterAppManager.java

```diff
@@ -200,7 +200,8 @@ void stopApp(@NotNull FlutterApp app) {
     AppStop appStop = new AppStop(app.appId());
     Method cmd = makeMethod(CMD_APP_STOP, appStop);
     sendCommand(app.getController(), cmd);
-    FlutterJsonObject obj = waitForResponse(cmd);
+    // We don't wait for a response here; we'll get an app.stop event later as a notification.
+    //waitForResponse(cmd);
```

Comment:
In my (one) test it did - we do sent the `stop` request, we just don't wait here for the response.